assisted-events-scrape/tests/integration/test_integration.py

```diff
@@ -1,17 +1,24 @@
 from config import ElasticsearchConfig
 import elasticsearch
 import waiting
-import time
 import pytest
 
-INIT_DELAY_SECONDS = 10
 ALL_EVENTS_NUMBER = 101
 
 
 class TestIntegration:
     def test_all_docs(self, _wait_for_elastic):
-        documents_count = self._es_client.count(index=self._config.index)['count']
-        assert(documents_count == ALL_EVENTS_NUMBER)
+        expected_count_idx = {
+            self._config.index: ALL_EVENTS_NUMBER,
+            ".events": ALL_EVENTS_NUMBER,
+        }
+        documents_count = 0
+        # As elasticsearch is eventually consistent, make sure data is synced
+        self._es_client.indices.refresh(index=self._config.index)
```

Comment:
This call makes sure the data is synced: this way we got rid of the timeout which could make the test flaky anyway.

Reply:
very nice!